frontends/web/src/routes/account/account.tsx

```diff
@@ -272,6 +273,15 @@ class Account extends Component<Props, State> {
                 })
             ) : '';
 
+        let offlineErrorTextLines = [];
```

Comment:
The linter wants `const` here, const because the reference to the array never changes.

Reply:
i `const`antly fall into this issue ;)